ext/pdo_sqlite/tests/subclasses/pdosqlite_003.phpt

```diff
@@ -0,0 +1,67 @@
+--TEST--
+PdoSqlite load extension
+--EXTENSIONS--
+pdo
+--FILE--
+<?php
+
+$db = Pdo::connect('sqlite::memory:');
+
+if (!$db instanceof PdoSqlite) {
+    echo "Wrong class type. Should be PdoSqlite but is [" .get_class($db) . "\n";
+}
+
+$extension_location = null;
+$locations = [
+    '/usr/lib/aarch64-linux-gnu/mod_spatialite.so',
+    "/usr/lib/x86_64-linux-gnu/mod_spatialite.so"
```

Comment:
On Alpinelinux it living in `/usr/lib/mod_spatialite.so`

Reply:
ok, thanks.